app/controllers/api/v1/users_controller.rb

```diff
@@ -4,6 +4,7 @@ class UsersController < ApiController
       include Api::UsersController
 
       before_action :authenticate!
+      before_action :authorize_super_admin, only: [:suspend]
```

Comment:
[Question] Why is authorization taking place in the controller versus the concern as with Articles unpublish (https://github.com/forem/forem/blob/main/app/controllers/concerns/api/articles_controller.rb#L109)?

Reply:
Ah, same concern as https://github.com/forem/forem/pull/18058/files#r917982864